acceptance/setup/pbs_package/pre-suite/072_install_puppet_agent.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+#require 'beaker-puppet'
+
+test_name "Install puppet agent" do
+  step "on the bolt controller" do
+    install_puppetlabs_release_repo_on(bolt, "puppet6-nightly")
+    bolt.install_package('puppet-agent')
+    result = on(bolt, "puppet agent --version")
+    assert_match(/6.0/, result.output)
+  end
+
+  step "restart puppet-agent service to generate certs" do
+    on bolt, puppet("agent -t"), :acceptable_exit_codes => [0,1]
```

Comment:
I'm really struggling to get the certificates working here. This is the output from `ls -R`:
```
    /etc/puppetlabs/puppet/ssl/ca/private:
    ca.pass
    
    /etc/puppetlabs/puppet/ssl/ca/requests:
    
    /etc/puppetlabs/puppet/ssl/ca/signed:
    
    /etc/puppetlabs/puppet/ssl/certificate_requests:
    gpdwbp3mpywyt9b.delivery.puppetlabs.net.pem
    
    /etc/puppetlabs/puppet/ssl/certs:
    ca.pem
    
    /etc/puppetlabs/puppet/ssl/private:
    
    /etc/puppetlabs/puppet/ssl/private_keys:
    gpdwbp3mpywyt9b.delivery.puppetlabs.net.pem
    
    /etc/puppetlabs/puppet/ssl/public_keys:
    gpdwbp3mpywyt9b.delivery.puppetlabs.net.pem
```

But when I go to sign the certificate it says there's no request.